quickstarts/singlestore/config.yml

```diff
@@ -0,0 +1,33 @@
+slug: singlestore
+description: |
+  ## About this integration
+    This integration uses an OpenTelemetry(OTeL) collector to gather metrics from [SingleStore's management API](https://docs.singlestore.com/cloud/reference/management-api/reference/) and visualize them in New Relic.
+
+  ## What does this integration cover?
+    This integration gives you complete visibility into your resource utilization and workload statistics for your Singlestore Database.
+
+summary: |
+  Monitor the performance of your SingleStore database using OTeL.
+icon: logo.svg
```

Comment:
Logo will bot visible under white background, kindly update logo

Reply:
@RamanaReddy8801 , I swapped it out for one with a clear background. Let me know if that works!

Reply:
@jcountsNR, 
That's fine, please make the change in the datasource folder as well.

Reply:
@RamanaReddy8801 done, let me know if there is anything else missing here.